source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
 
-If you anticipate syncing a large data set, or if you plan to pause
-synchronization for an extended period of time, increase the size of the
-replica set ``oplog`` in the source cluster.
+
+Removing older operations from the oplog of the source cluster   
+before ``mongosync`` applies them to the destination cluster
+causes the migration to fail and ``mongosync`` to exit.
+
+During initial sync, ``mongosync`` may apply operations at a slower
```

Comment:
Some typos:

During initial sync, ``mongosync`` may apply operations at a slower
rate. After ``mongosync`` completes the initial sync, it applies changes faster and stays more current in the source cluster ``oplog``.